app/messages/Messages.scala

```diff
@@ -545,6 +546,11 @@ object Messages {
       english = "Select yes if you are the account holder"
     )
 
+    // BARs
+    val accountNumberNotWellFormatted: FormError = FormError("bars", "validate.accountNumberIsWellFormatted.no")
```

Comment:
you may need to do something in the views template for this page since the href of the link in the error summary is based on the key of the error but there's won't be a field with id `bars` at the moment. 

You may have to link to a specific field depending on the bars error, e.g. `accountNumberNotWellFormatted` with the account number field, `sortCodeNotPresentOnEiscd` with the sort code field etc. 

We also need to check that the error message displays above the appropriate field entry box on the page

Reply:
> you may need to do something in the views template for this page since the href of the link in the error summary is based on the key of the error but there's won't be a field with id `bars` at the moment.
> 
> You may have to link to a specific field depending on the bars error, e.g. `accountNumberNotWellFormatted` with the account number field, `sortCodeNotPresentOnEiscd` with the sort code field etc.
> 
> We also need to check that the error message displays above the appropriate field entry box on the page

I discussed with Karen about initially not linking the errors to actual fields as with these errors it is not known which field (sortCode or accountNumber) has the error. So by using field `bars` we end up with just the error summary and an empty href.

With `accountNumberNotWellFormatted` it may be the sortCode that is wrong and with `sortCodeNotPresentOnEiscd` it may be that the accountNumber is wrong. 

So this is deliberate, and a first pass. Not sure if it is correct behaviour though. Maybe we can chat with Karen?

Reply:
> these aren't actual messages per se so it doesn't feel like they belong here

I'll think about where they can go...

Reply:
the error summary message needs to point to somewhere according to the design system here:
https://design-system.service.gov.uk/components/error-summary/
I think part of that may be for accessibility reasons so users can navigate to the relevant area quickly if they're using a screen reader. Where it's uncertain I remember linking to the first relevant box that could be wrong, so on a date if you don't know what was wrong then you'd point to the day field I think. Worth a double check, yeah

Reply:
actually it does say this on that page so what I said about linking to the first thing that could be wrong is correct I think:
```
If you do not know which field contains an error, link to the first field.
```